Lib/test/test_doctest.py

```diff
@@ -3042,7 +3042,7 @@ def test_CLI(): r"""
     >>> print(normalize(err))                    # doctest: +ELLIPSIS
     Traceback (most recent call last):
       ...
-    FileNotFoundError: [Errno ...] No such file or directory: 'nosuchfile'
+    FileNotFoundError: [Errno ...] ...nosuchfile...
```

Comment:
Would it be possible to add a comment explaining that the exact error message changes depending on the platform?

Reply:
Added.